pyproject.toml

```diff
@@ -1,17 +1,35 @@
 [tool.poetry]
 name = "nautobot-version-control"
-version = "0.1.0"
+version = "1.0.0-alpha.0"
 description = "Nautobot Version Control"
 authors = ["Network to Code, LLC <opensource@networktocode.com>"]
-license = "LICENSE.txt"
-
+license = "Apache-2.0"
+repository = "https://github.com/nautobot/nautobot-plugin-version-control"
+documentation = "https://nautobot-version-control.readthedocs.io"
+readme = "README.md"
+classifiers = [
+    "Intended Audience :: Developers",
+    "Development Status :: 3 - Alpha",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+]
 packages = [
     { include = "nautobot_version_control" },
 ]
+include = [
+    "LICENSE.txt",
+]
 
 [tool.poetry.dependencies]
 python = "^3.6.2"
-nautobot = "~1.2.0b1"
+# TODO(jathan): Pin develop for now because it has upstream fixes that are not yet released. Once
+# v1.2.0 final is out move this to the next version.
+# nautobot = "~1.2.0b1"
+nautobot = { git = "https://github.com/nautobot/nautobot.git", branch = "develop" }
```

Comment:
Actually, isn't our current best practice to have pyproject.toml (and especially poetry.lock) not specify a nautobot dependency at all? When installing the plugin we should already *have* the right version of nautobot installed (whether locally, or via the baseline docker container we're using) and this makes it possible (even likely) that installing the plugin will inadvertently upgrade or even downgrade nautobot as an unintended side effect.

Reply:
There has been contention here but I am pretty firm on Nautobot always being a hard dependency of plugins that way it becomes impossible to introduce dependency conflicts with core.

Reply:
It's mostly an issue of local development + `poetry.lock` I suppose - by having this in `poetry.lock` any development environment for this plugin will install the exact commit from GitHub that is pinned in poetry.lock, even if a newer/better version of Nautobot is available, and overriding any Nautobot version provided by the baseline Docker image.